docs/src/main/sphinx/conf.py

```diff
@@ -128,6 +129,7 @@ def setup(app):
 html_title = '%s %s Documentation' % (project, release)
 
 html_logo = 'images/trino.svg'
+html_baseurl = 'https://trino.io/docs/current/'
```

Comment:
Is this hardcoding the URL in the output of something specifically or everywhere? Ideally this is only used in the sitemap or otherwise limited so that the generated binaries are still good to reuse on other server

Reply:
it's already hardcoded elsewhere in the config: 
```
html_theme_options = {
    'base_url': 'https://trino.io/docs/current/',
```